src/main/java/com/accantosystems/stratoss/vnfmdriver/service/LifecycleManagementService.java

```diff
@@ -109,4 +107,14 @@ public ExecutionAcceptedResponse executeLifecycle(ExecutionRequest executionRequ
         }
     }
 
+    private String getStringRequestProperty(ExecutionRequest executionRequest, String propertyName) throws MessageConversionException {
```

Comment:
Two comments on this one.
1. Would this be better as a helper function on the ExecutionRequest class?
2. Is it valid to throw an error if it's not a StringPropertyValue? Is it valid for the caller to send null here if its optional?